HW1/P6/P6.py

```diff
@@ -0,0 +1,77 @@
+import re
+import random
+
+def clean(s):
+    regPattern = ['(^[0-9]+$)', '^[A-Z]+$', '^[A-Z]+\.$']
+    regObj = [re.compile(r) for r in regPattern]
+
+    ret = []
+    s = s.strip().split()
+    for w in s:
+        w = w.strip()
+        rTests = [(r.match(w) == None) for r in regObj]
+        if False not in rTests:
+            ret.append(w)
+    return ret
+
+# clean("123 123. foo bar. BAR. 1 ABC")
+
+def sepTriple(((k),v)):
+    print k, v
+    return ((k[0], k[1]), [(k[2],v)])
+
+print 'Cleaning input...'
+
+allWords = []
+with open('pg100.txt','r') as f:
+    for line in f:
+        allWords += clean(line)
+
```

Comment:
Same comment as above: did you consider using RDDs for this part of the code as well?